kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/configuration/writer/DhcpServerConfigWriterTest.java

```diff
@@ -347,14 +424,32 @@ private void thenKuraExceptionIsCaught() {
         assertTrue(this.isKuraException);
     }
 
+    /*
+     * Utilities
+     */
+
     @After
-    public void clean() {
-        File dhcpConfigFile = new File("/tmp/dhcpd-eth0.conf");
-        if (dhcpConfigFile.exists()) {
-            dhcpConfigFile.delete();
+    public void cleanUp() throws IOException {
+        dhcpServerMock.close();
+        this.mockFiles.delete();
+        this.mockFiles.create();
+    }
```

Comment:
Given how JUnit works I don't think you need these....

Reply:
The static mock needs the close here. For the tmp files I will remove those lines.